tests/integration/test_hive_query/configs/config.xml

```diff
@@ -13,7 +13,7 @@
 
     <local_cache_for_remote_fs>
         <enable>true</enable>
-        <root_dir>/tmp/clickhouse_local_cache</root_dir>
+        <root_dir>/tmp/clickhouse_local_cache,/tmp/clickhouse_local_cache1</root_dir>
```

Comment:
Is it possible to add an integration test?

Reply:
Do you mean to change the path list ?
I don't find a way to restart a container in the intergration.

Reply:
I mean to check in test that both paths are used, which can be done with the help of `exec_in_container` from helpers/cluster.py